pkg/util/version/const.go

```diff
@@ -64,7 +64,7 @@ var (
 
 // FluentbitImage contains the location of the Fluentbit container image
 func FluentbitImage(acrDomain string) string {
-	return acrDomain + "/fluentbit:1.7.8-1"
+	return acrDomain + "/fluentbit:1.9.9-1"
```

Comment:
should this be `1.9.10` like we have in the makefile

Reply:
No, since this should build it and such but not activate it in production yet.